boards/DCM/Src/App/states/App_SharedStates.c

```diff
@@ -31,3 +31,49 @@ void App_SharedStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     App_CanTx_SetPeriodicSignal_LEFT_INVERTER_SWITCH(
         can_tx, App_InverterSwitches_IsLeftOn(inverter_switches));
 }
+
+void App_SharedStates_ConfigInverterSwitches(struct DcmWorld *world)
+{
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct InverterSwitches * inv_switches =
+        App_DcmWorld_GetInverterSwitches(world);
+
+    // Close or open the inverter LV switches if requested by a PCAN node
+    if (App_CanRx_INV_LOAD_SWITCHES_CMD_GetSignal_CLOSE_LEFT_SWITCH(can_rx) ==
+        CANMSGS_INV_LOAD_SWITCHES_CMD_CLOSE_LEFT_SWITCH_YES_CHOICE)
```

Comment:
We can maintain the enum for readability within PCAN, but for these checks I think it looks cleaner to do

```suggestion
    if (App_CanRx_INV_LOAD_SWITCHES_CMD_GetSignal_CLOSE_LEFT_SWITCH(can_rx) )
```

Reply:
with this implemented as an if-else it should look much cleaner

Reply:
done